src/cli/trn_cli_xdp.c

```diff
@@ -149,9 +149,10 @@ int trn_cli_load_transit_subcmd(CLIENT *clnt, int argc, char *argv[])
 	char rpc[] = "load_transit_xdp_1";
 	char xdp_path[1024];
 	char pcapfile[1024];
+	int xdp_flag = 2;
```

Comment:
Is this for "generic" (XDP_FLAGS_SKB_MODE)?

Reply:
Yeah, let me change it to the defined constant